server/crashmanager/views.py

```diff
@@ -1616,7 +1616,19 @@ class BugzillaTemplateEditView(UpdateView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+
+        if "provider" in self.request.GET:
+            provider = get_object_or_404(BugProvider, pk=self.request.GET["provider"])
+        else:
+            user = User.get_or_create_restricted(self.request.user)[0]
+            provider = get_object_or_404(BugProvider, pk=user.defaultProviderId)
+
         context["title"] = "Edit template"
+        context["template_id"] = self.kwargs[self.pk_url_kwarg]
+        context["provider"] = provider
+        context["mode"] = (
+            "Bug" if self.object.mode == BugzillaTemplateMode.Bug else "Comment"
```

Comment:
Minor nit but `BugzillaTemplateMode.Bug == "bug"` vs `"Comment"`.  I'd prefer `BugzillaTemplateMode.Comment` for consistency.